src/main/java/ca/wolfram/beta/symath/operations/MultiplyNode.java

```diff
@@ -45,7 +91,15 @@ public double operationEval(VMap map) {
     public String toString() {
         StringBuilder s = new StringBuilder();
         s.append("(");
-        getChildren().forEach((c) -> s.append(c).append(" * "));
-        return s.substring(0, s.length() - 3) + ")";
+        boolean isFirst = true;
+        for (MathNode c : getChildren()) {
+            if (isFirst) {
+                s.append(c.toString());
+                isFirst = false;
+                continue;
+            }
+            s.append(" * ").append(c.toString());
+        }
```

Comment:
I would still prefer if you switch over to my code so we don't need `continue` and also i think we can remove the if check in my code and just assume it is correct.